packages/core/src/lib/Slider/Slider.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import { ISliderProps } from './types';
+import styles from './Slider.styles';
+
+const { sliderDiv, sliderInput, sliderLabel, sliderP } = styles;
+
+const Slider: React.FC<ISliderProps> = ({
+    rangeId,
+    rangeLabel,
+    rangeMax,
+    rangeMin,
+    rangeValue,
+}) => {
+    const [currentValue, setCurrentValue] = useState(rangeValue);
+
+    const rangeChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const newRangeValue = Number(event.currentTarget.value);
+        setCurrentValue(newRangeValue);
+    };
+
+    return (
+        <styles.sliderDiv>
+            <styles.sliderLabel htmlFor={rangeId}>
+                {rangeLabel}
+            </styles.sliderLabel>
+            <styles.sliderP>
+                {currentValue} of {rangeMax}
+            </styles.sliderP>
+            <styles.sliderInput
```

Comment:
Range label and this displayed value can be optional....

Reply:
Would you mind explaining more on this,like what other than range label

Reply:
Label and the displayed value i.e `{currentValue} of {rangeMax}`
User may not want to show it this way....
Instead you can make this optional using a new type probably.. if user wishes to see the message they can.
This can be added in future iterations so not a big blocker.

Reply:
Removing label at the moment,label could be made by using onChange callback and reieveing the value

Reply:
Would implement a label props later to takle this